Homework5/2.py

```diff
@@ -0,0 +1,141 @@
+#Создайте программу для игры с конфетами человек против человека.
+
+import os
+import random
+
+def clear():
+    os.system('clear')
```

Comment:
Проверяйте пожалуйста, платформу, прежде чем использовать модуль `os`. На windows другая команда
```suggestion
def clear():
    if os.name == 'nt':  # это windows
        os.system('cls')
    else:
        os.system('clear')  # не уверен, что подойдет для всех остальных систем, но для Linux точно.
```
Можно также использовать модуль `sys` и функцию `platform`, либо модуль `platform` и функцию `system`